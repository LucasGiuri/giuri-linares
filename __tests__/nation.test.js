import React from 'react';
import renderer from 'react-test-renderer';

import Nation from '../pages/nation.js';
jest.mock('../components/Content/Content', () => 'Content');

describe('Nation', () => {
  it('Nation basic render', () => {
    const component = renderer.create(<Nation />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
})
