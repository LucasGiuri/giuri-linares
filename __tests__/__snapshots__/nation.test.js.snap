// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nation Nation basic render 1`] = `
Array [
  <h1>
    Nation !
  </h1>,
  <Content />,
]
`;
